client/app/utils/users/adminUserOperators.ts

```diff
@@ -29,50 +27,24 @@ async function getAdminsApprovedUserOperators(): Promise<UserOperator[]> {
   }
 }
 
-// 🛠️ Function to fetch userOperators
```

Comment:
This file was written with the assumption that an industry user could access more than one operator. We're keeping separate app_role and UserOperator.role in the database for future flexibility, but in the app, we're going with one operator per user. So I've simplified these helper functions since they no longer need to handle the case of multiple operators per user.